app/src/main/java/com/jerboa/ui/components/post/PostActivity.kt

```diff
@@ -309,7 +309,7 @@ fun PostActivity(
                     is ApiState.Failure -> ApiErrorText(postRes.msg, padding)
                     is ApiState.Success -> {
                         val postView = postRes.data.post_view
-                        val moderators = postRes.data.moderators
+                        val moderators = remember(postRes) { postRes.data.moderators.map { it.moderator.id } }
```

Comment:
You have a remember for moderators both in the postlisting, and here. One can probably be removed.

Reply:
Those remember different things?